apps/core/components/ItineraryCard/ItineraryCardWrapper.web.js

```diff
@@ -18,13 +18,20 @@ import {
 type Props = {|
   +localizedPrice: string,
   +children: React.Node,
+  +detailOpened?: boolean,
   +layout: number,
 |};
 
-const noop = () => {}; // @TODO
+const noop = () => {};
```

Comment:
We have this function exported from `@kiwicom/margarita-utils` so we don't have to create it everywhere 😊

Reply:
Nice, updated.